src/ringmesh/geogram_extension/geogram_mesh.cpp

```diff
@@ -76,5 +77,9 @@ namespace RINGMesh
         VolumeMeshBuilderFactory3D::
             register_creator< GeogramVolumeMeshBuilder3D >(
                 GeogramVolumeMesh3D::type_name_static() );
+        std::cout << "je suis a la fin " << std::endl;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of "::std::cout" by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS106)